GVRf/Sample/gvr-video/src/org/gearvrf/video/VideoScript.java

```diff
@@ -410,6 +410,12 @@ public void run() {
                     mGVRContext, "button/play-active.png"));
             mPlayPauseButton = new GVRSceneObject(gvrContext,
                     gvrContext.createQuad(0.7f, 0.7f), mInactivePause);
+            mPlayPauseButton.getRenderData().getMaterial().setTexture("active_play", mActivePlay);
```

Comment:
Either way is not clean for me. Discussion on changing the ref count mechanism as a permanent solution is going on a separate thread, where I put my ideas there.